tests/testthat/test_chainof5parts.R

```diff
@@ -77,8 +77,8 @@ test_that("chainof5parts", {
   expect_equal(round(mean(IMP$metashort$ENMO), digits = 5), 0.02911, tolerance = 3)
   expect_that(round(as.numeric(SUM$summary$meas_dur_dys), digits = 5), equals(1.95833))
   expect_that(ncol(SUM$daysummary), equals(34))
-  expect_equal(SUM$daysummary$`p50_ENMO_mg_0-24hr`, c("17.15", "33",  "0"))
-  expect_equal(round(as.numeric(SUM$daysummary$`p90_ENMO_mg_0-24hr`)), c(44, 54, 41), tolerance = 0)
+  expect_equal(SUM$daysummary$`p50_ENMO_mg_0-24hr`, c("0", "32.65",  "0"))
```

Comment:
Why do we have to change the percentile in the unit-test line 80 and 81? Does this mean a bug was fixed and the old output was not correct?

Reply:
This came from the change to find duplicates (rounding to 3 decimals) in g.calibrate. Since it did not improve the calibration, I have reset g.calibrate as in master.